geotrellis-python/README.md

```diff
@@ -0,0 +1,43 @@
+GeoTrellis uses [Apache Avro](https://avro.apache.org/) serialization when storing its tiled map layers. While GeoTrellis is a Scala project Avro provides libraries for multiple languages, including python. This project is to implement a python library that will read and write persisted GeoTrellis layers and present them as [NumPy](http://www.numpy.org/) datatypes that can be used by [rasterio](https://github.com/mapbox/rasterio). Once the basic translation is supported the next step would be to read GeoTrellis layer in a distributed Python Spark job and use rasterio to perform geospatial computations.
+
+Interoperability between GeoTrellis and rasterio will allow for complex, multi-stage geospatial workflows that do not suffer the usual performance penalties when working mixed Spark Scala/Python environment, expanding the utility of both libraries.
+
+#### Dependencies
+
+- [Apache Avro](https://avro.apache.org/)
+- [Avro Json Serializer](https://github.com/linkedin/python-avro-json-serializer)
+- [NumPy](http://www.numpy.org/)
+- [rasterio](https://github.com/mapbox/rasterio)
+- [pytz](https://github.com/newvem/pytz)
+- [pypng](https://github.com/drj11/pypng)
+- [Spec](https://github.com/bitprophet/spec) for testing
```

Comment:
When I do a `pip install spec` on my machine, I get `spec version 1.3.7`. When I run the `spec --with-inherited --with-tools-decorators-only` below, I get

```console
Usage: spec [options]

spec: error: no such option: --with-inherited
```

If I run spec without --with-inherited, I get 

```console
Usage: spec [options]

spec: error: no such option: --with-tools-decorators-only
```

if I run just `spec`, I get failures like
```console
TypeError: __init__() takes exactly 2 arguments (1 given)
```